test/e2e/syncer/wildwest.dev_cowboys.yaml

```diff
@@ -0,0 +1,58 @@
+
+---
```

Comment:
Why did this need to get copied?

Reply:
It doesn't, I can refer to the under `reconciler/cluster`.

Reply:
ah, well it looks like [I can't use `..` in a `go:embed` directive](https://pkg.go.dev/embed):

> Patterns may not contain ‘.’ or ‘..’ or empty path elements, nor may they begin or end with a slash.

Reply:
yes that makes sense.  Mind if I make this a follow up issue? I'm thinking we should move the CRD file and gen'd packages up under something like `test/e2e/crds`, update `hack/update-codegen.sh`, and expose some kind of interface for pulling in CRDs into a test.

Reply:
This is no longer needed since I removed the syncer e2e tests.  The cluster controller e2e tests were already testing everything the syncer e2e tests were.